src/client/libcurve_file.h

```diff
@@ -54,34 +54,35 @@ class LoggerGuard {
     static void InitInternal(const std::string& confpath);
 };
 
-// FileClient是vdisk的管理类，一个QEMU对应多个vdisk
+// FileClient is the management class of vdisk,
+// a QEMU corresponds to multiple vdisks
 class FileClient {
  public:
     FileClient();
     virtual ~FileClient() = default;
 
     /**
-     * file对象初始化函数
-     * @param: 配置文件路径
+     * initialization
```

Comment:
sorry for my late fix (for midterm exam...), but it seems `@xxx xxx` (without ':') would be better according to the code in `src/mds`, and it would also not work well in my local vscode if ':' added, please take a look if free.

Reply:
> sorry for my late fix (for midterm exam...), but it seems `@xxx xxx` (without ':') would be better according to the code in `src/mds`, and it would also not work well in my local vscode if ':' added, please take a look if free.

yeap, it's my oversight, the correct way to use keyword `@brief` is as you said :)